script.js

```diff
@@ -0,0 +1,52 @@
+let modalShow = document.querySelector("#modal-show");
+let modalClose = document.querySelector("#modal-close");
+let modalSuccess = document.querySelector("#modal-success");
+let modal = document.querySelector("#modal")
```

Comment:
Try using const instead of let

Reply:
I don't think it might create any difference but still thanks for the suggestion. 😄